htslib/vcf.h

```diff
@@ -1528,6 +1556,86 @@ static inline int bcf_format_gt(bcf_fmt_t *fmt, int isample, kstring_t *str)
     return e == 0 ? 0 : -1;
 }
 
+/**
+ *  bcf_format_gt1 - formats GT information on a string
+ *  @param hdr - bcf header, to get version
+ *  @param fmt - pointer to bcf format data
+ *  @param isample - position of interested sample in data
+ *  @param str - pointer to output string
+ *  Returns 0 on success and -1 on failure
+ *  This method is extended from bcf_format_gt to output phasing information
+ * in accordance with v4.4 format, which supports explicit / prefixed phasing
+ * for 1st allele.
+ * Explicit / prefixed phasing for 1st allele is used only when it is a must to
+ * correctly express phasing.
+ */
+static inline int bcf_format_gt1(const bcf_hdr_t *hdr, bcf_fmt_t *fmt, int isample, kstring_t *str)
```

Comment:
The suffix 1 as it is used here is confusing and somewhat conflicting with the convention used in the rest of the code. Can you please rename to include the 'phase' keyword? Or, create an extended function where various flags can be turned on, starting with PHASE_1ST_ALLELE, or something like that

Reply:
I think the name should not be made more specific, and if possible made more generic!

An implementation which supports the newer format should be the 1st choice going forward.
Best would have been to update the current method itself but not possible as it would break the compatibility due to signature change.
Hence chose a name which is in sync with current method. Also we have a few others like index_load/index_build/ etc in same lines.

Reply:
updated the name to bcf_format_gt_v2